interface/login/login.php

```diff
@@ -1,248 +1,247 @@
 <?php
 /**
- * Login screen.
- *
- * LICENSE: This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * as published by the Free Software Foundation; either version 2
- * of the License, or (at your option) any later version.
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
- *
- * @package OpenEMR
- * @author  Rod Roark <rod@sunsetsystems.com>
- * @author  Brady Miller <brady@sparmy.com>
- * @author  Kevin Yeh <kevin.y@integralemr.com>
- * @author  Scott Wakefield <scott.wakefield@gmail.com>
- * @author  ViCarePlus <visolve_emr@visolve.com>
- * @author  Julia Longtin <julialongtin@diasp.org>
- * @author  cfapress
- * @author  markleeds
- * @link    http://www.open-emr.org
- */
-
-$fake_register_globals=false;
-$sanitize_all_escapes=true;
-
-$ignoreAuth=true;
+* Login screen.
+*
+* LICENSE: This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+* You should have received a copy of the GNU General Public License
+* along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+*
+* @package OpenEMR
+* @author  Rod Roark <rod@sunsetsystems.com>
+* @author  Brady Miller <brady@sparmy.com>
+* @author  Kevin Yeh <kevin.y@integralemr.com>
+* @author  Scott Wakefield <scott.wakefield@gmail.com>
+* @author  ViCarePlus <visolve_emr@visolve.com>
+* @author  Julia Longtin <julialongtin@diasp.org>
+* @author  cfapress
+* @author  markleeds
+* @link    http://www.open-emr.org
+*/
+
+$fake_register_globals = false;
+$sanitize_all_escapes = true;
+
+$ignoreAuth = true;
 include_once("../globals.php");
 include_once("$srcdir/sql.inc");
+
+$cookieValue = session_name() . " = " . session_id();
```

Comment:
When I was looking over that function I found it very difficult to work with it given the interpolation of PHP and JS so I moved the PHP out and only reference it via echo. It helps keep the code cleaner, in my opinion. Something I'd like to see more of